src/main/java/org/aludratest/eclipse/vde/internal/editors/VisualDataEditorPage.java

```diff
@@ -379,6 +379,22 @@ private IInputValidator createConfigNameValidator(boolean edit) {
 
 		return new ConfigNameValidator(disallowedNames);
 	}
+	
+	private IInputValidator createExternalTestIDValidator() {

```

Comment:
This validator does not really make sense. It just checks that the same external test ID is not reused within the same test data file (but not over all test data files, of course). 
I think that the external test ID input should not be validated at all.